libs/wingcompiler/project.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "compiler",
```

Comment:
I'm concerned we'll have some naming confusion.

`wingc` is technically `wingcompiler`. I would argument it would be more accurate to say that `wingc` should be called `wing-frontend` or `wing-compiler-frontend`.

Reply:
Right. I chose `compiler` because the `libs/wingsdk` package is named `sdk` in the `project.json` file.

So, should I rename `wingc` to `wing-compiler-frontend`?

Reply:
Discussed in https://winglang.slack.com/archives/C048QCN2XLJ/p1686137164071349

No rename needed